tools/util/vfio.sh

```diff
@@ -11,15 +11,15 @@ modprobe vfio-pci
 for file in $(find /sys/devices/ -name *sriov_totalvfs*); do
     pfroot=$(dirname $file)
 
-    # enable all available VFs
-    cat $file > $pfroot/sriov_numvfs
+    # enable all available VFs. If this fails we skip the device continuing with the others.
```

Comment:
Can you please add a warning if the enable VFs fails so it will be better for the user to know there was an issue when he run the script.

lets not silent errors :)